cmds/webboot/webboot.go

```diff
@@ -176,6 +177,36 @@ func (d *DownloadOption) exec(uiEvents <-chan ui.Event, network bool, cacheDir s
 
 	}
 
+	// verify checksum if the distro chosen is supported
```

Comment:
Start the comment with capital letter
```suggestion
	// Verify checksum if the distro chosen is supported
```